### ISSUE TITLE
Two Player for PS4 Ports

### ISSUE THREAD

**Issue by davak** [Cpasjuste/pemu#225]
Incredible work!  I am running all three PS4 ports. I am happy to donate! 

I don't see any way to enable player 2 on any of the PS4 ports including NES, SNES, and PFBneo.  For example, I don't see a player 2 start on PFBneo and I can't get player 2 dualshock to work on NES and SNES ports. 

Thank you in advance.  Your work thus far is incredible!

**Comment by davak** [comment]
You are absolutely correct!   You just have to have both connectors fully connected before launch.   I thought I checked that.  I'm sorry. 

The only other interesting wrinkle is for PFBneo the 2 player start is L1 on the second controller.  In some arcade games where there is a traditional single joystick (Ms. PacMan), that's the only time the second controller is used.   Not a big issue at all, but I'll spell it out here so google can help anybody else. 

You just made my year.

**Comment by HLUK2022** [comment]
I don't see any way to enable player 2 PGen on
PS4.
